### ISSUE TITLE
Format `compat_features` with BCD's sort order

### ISSUE THREAD

**Issue by ddbeck** [web-platform-dx/web-features#834]
Right now, if you generate a dist file from a `compat_features` override, then the order of keys can be different from the order given by traversing features from BCD. In other words, if you remove the override, the dist can change the order of `compat_features`, even if the set of keys has not.

To make material changes in feature lists more apparent, we should enforce a key order. It'd be nice to have a formatting script that can check for or write format changes (e.g., with options for checking for CI and writing for authors). The sort order ought to be the same as BCD's order (e.g., as given by `npm run traverse` in BCD).

**Comment by captainbrosset** [comment]
@ddbeck to verify if this should be closed.

**Comment by ddbeck** [comment]
I don't think this matters any more. Or, at least, it's not so apparent now that there's less churn on features. I think we ought to close this, though if it comes up again and it's needed after all, we may refile or reopen.
